content/en/blog/2024/authz-policy-with-kyverno/index.md

```diff
@@ -0,0 +1,469 @@
+---
+title: Policy based authorization - Using Kyverno Authz Server
+description: Delegate authorization decisions logic to Kyverno Authz Server, leveraging Kyverno authorization policies based on CEL bringing safety, flexibility, and unprecedented performance.
```

Comment:
```suggestion
title: title: Policy based authorization using Kyverno
description: Delegate Layer 7 authorization decision logic using Kyverno's authz server, leveraging policies based on CEL.
```

It's unclear from the docs & GitHub page for your Envoy plugin if you want "Kyverno Authz Server" to be a Brand Name throughout.  If you don't, I'd recommend you change to "Kyverno authz server" throughout the doc.

Reply:
My feeling is that "Kyverno Authz Server" is better to avoid confusion with Kyverno (the policy engine).